docs/blocks/head-like-rotation.md

```diff
@@ -0,0 +1,506 @@
+---
+title: Specific (Head-Like) Rotation
+category: Tutorials
+tags:
+    - experimental
+    - expert
+mention:
+    - uspel
+    - QuazChick
+---
+
+
+This tutorial guides you through making a block with proper head-like rotation (e.g. creeper heads and signs), providing examples of a "shell" block with this rotation type.
+
+:::warning BEGINNERS BEWARE!
+
+This tutorial assumes good knowledge of [blocks 1.16.100+](/blocks/blocks-16), [advanced Molang](/documentation/advanced-molang) and creating models in applications such as [Blockbench](https://blockbench.net).
+
+:::
+
+![Custom shell blocks oriented randomly](/assets/images/blocks/head-like-rotation/showcase.png)
+
+Features:
+
+-   Can be placed on the top of blocks, with 16 possible rotations
+-   Can be placed on the side faces of blocks (north, east, south & west)
+-   Rotation behaves the same as vanilla mob heads - without the performance hit from block entities!
+
+Experiments:
+
+- Holiday Creator Features
+- Upcoming Creator Features
+
+## Block Model
+
+To allow for more specific rotation, your block's model will need a few extra bones.
+
+There are 4 bones required for specific ground rotation, each with different Y axis rotations:
+
+-   `up0` (Y rotation = 0)
+-   `up1` (Y rotation = 22.5)
+-   `up2` (Y rotation = 45)
+-   `up3` (Y rotation = 67.5)
+
+**These rotations are in a clockwise direction.**
+
+The bones will likely be duplicates of each other, excluding rotation change.
+
+:::tip
+
+Keep your bones' pivots set to `[0, 0, 0]` so that their rotation is around the middle of the block.
+
+:::
+
+In addition, a `side` bone will be necessary for placement on side faces.
+
+
+The following model for a "shell" block can be used as a reference:
+
+![](/assets/images/blocks/head-like-rotation/model_bones.png)
+
+<Spoiler title="Shell Example Model">
+
+<CodeHeader>RP/models/blocks/shell.geo.json</CodeHeader>
+
+```json
```

Comment:
The geometry bones should have a more descriptive names, from my add-on for example, I used head_45 to notate what angle it is facing